File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int ans = 0;
        for(int i=0;i<nums.length;i++){
            if(search(nums[i])){
                ans+=1;
            }
        }
        return ans;
    }
    boolean search(int num){
        int len = 0;
        int temp = num;
        while(temp>0){
            len+=1;
            temp = temp/10;
        }
        if(len%2 == 0){
            return true;
        }
        return false;
    }
}

File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        ans = 0
        for num in nums:
            count = 0
            while num>0:
                num = num//10
                count+=1
            
            if count%2 == 0:
                ans+=1
        
        return ans